apps/namadillo/src/hooks/useTransactionNotifications.tsx

```diff
@@ -106,6 +107,28 @@ export const useTransactionNotifications = (): void => {
     });
   });
 
+  useTransactionEventListener("Bond.PartialSuccess", (e) => {
+    const { id, total } = parseTxsData(e.detail.tx, e.detail.data);
+    clearPendingNotifications(id);
+    dispatchNotification({
+      id,
+      title: "Some staking transactions failed",
+      description: (
+        <>
+          Your staking transaction of <NamCurrency amount={total} /> has
+          succeeded
+        </>
+      ),
+      failedDescription: (
+        <>The following staking transactions were not applied:</>
+      ),
+      details: getAmountByValidatorList(e.detail.data),
+      failedDetails: getAmountByValidatorList(e.detail.failedData || []),
+      type: "partialSuccess",
+      forceDetailsOpen: true,
```

Comment:
an alternative for this is using the `description` field with a component, then removing the `failedDescription`, `failedDetails` and `forceDetailsOpen` ✂️ 

something like this:

```tsx
dispatchNotification({
  ...
  description: <PartialSuccessDescription detail={e.detail} />
})

const PartialSuccessDescription = (detail) => {
  return (
    <>
      <div className="w-full text-xs text-white block">
        <div>{detail.details}</div>
        <div>{detail.failedDescription}</div>
        <div>{detail.failedDetails}</div>
      </div>
    </>
  )
}

Reply:
Nice, I will do this! Thanks!

Reply:
Done!